docs/Release-Notes-2.16.md

```diff
@@ -0,0 +1,133 @@
+ERROR:     Project page: https://rockylinux.org/
+ERROR:     Enable permanently by:
+ERROR:     $ ln -s /etc/mock/rocky+epel-8-x86_64.cfg /home/praiskup/.config/mock/epel-8-x86_64.cfg
+```
+
+Additional changes are:
+
+ * Fedora 33 configs were moved to eol/ directory
+ * EOLed EPEL Playground configs
+ * Added configs for CentOS Stream 9 + EPEL Next 9
+ * We expanded `dnf_vars` which cause an issue on EL7 hosts [RHBZ#2026571](https://bugzilla.redhat.com/show_bug.cgi?id=2026571)
+ * Added compatibility symlinks for EPEL 7 to centos+epel-7-*
+ * Resolved the multiple "local" repo problems
+ * Dropped rhel+epel-8-ppc64 config
+ * Added rhel+epel-8-s390x config
+ * Added navy-8-x86_64 config
+ * Reduced packages installed in epel chroots
+
+## Mock 2.16 changes:
+
+Mock got a new configuration option:
+
+```
+config_opts["no-config"]["epel-8"] = {}
+config_opts["no-config"]["epel-8"]["alternatives"] = {
+    "alma+epel-8": {
+        "description": [
+            "Builds against AlmaLinux 8 repositories, "
+            "together with the official EPEL repositories.",
+            "Project page: https://almalinux.org/"
+        ],
+    }
+    }
+```
+
+When the configuration file for `epel-8-*` does not exist, it will print the text from the `description` field.
+There is new file `/etc/mock/chroot-aliases.cfg` which contains defaults, but you can add your own option in your user config.
+
+There was one issue with BSD Tar, which has been resolved [[GH#820](https://github.com/rpm-software-management/mock/pull/820)]
+
+There is new option `ssl_extra_certs` [[GH#801](https://github.com/rpm-software-management/mock/pull/801)]
+
+```
+config_opts['ssl_extra_certs'] = ['/etc/pki/tls/certs/client.crt', '/etc/pki/tls/certs/',
+                                  '/etc/pki/tls/private/client_nopass.key.crt', '/etc/pki/tls/private/']
+#config_opts['ssl_extra_certs'] = ['/path/on/host', '/path/in/mock/chroot',
+#                                  '/path/on/host2', '/path/in/mock/chroot2', ... ]
+```
+
+It copies the host's SSL certificates into a specified location inside the chroot if
+mock needs access to repositories requiring client certificate
+authentication. Specify the full path to the public certificate on the host
+and the destination directory in the chroot. Do the same for the private key.
+The private key should not be password-protected if you want Mock to run
+unattended. 
+
+This is the last 2.x release made from `main` branch. After this release, we
+will create a new branch, and future 2.x versions will get only important bug
+fixes and important changes to the config.
+
+A version in `main` will be 3.x and will stop supporting EL7 as *host*. This
+will allow us to get rid of some compatibility code. However, we will still
+support building **for** EL7.
+
+We plan to build 3.x for Fedora 35+ and EPEL 9+ (and after a few releases for
+EPEL 8 too). Fedora 34 and EPEL 7 will stay on 2.x version.
```

Comment:
Do you see any benefit in **not** shipping v3 into EL8 and F34?  At least I planed to batch
update all Fedora and EPEL 8+.

Reply:
I'm not sure there's a reason _not_ to update to v3 there...

Reply:
Ok, I will update this when doing the release, but thanks @xsuchy for writing those docs!